src/tests/contract_test/profile_test.cljs

```diff
@@ -0,0 +1,25 @@
+(ns tests.contract-test.profile-test
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    legacy.status-im.events
+    [legacy.status-im.multiaccounts.logout.core :as logout]
+    legacy.status-im.subs.root
+    status-im.events
+    status-im.navigation.core
+    status-im.subs.root
+    [test-helpers.integration :as h]
+    [tests.contract-test.utils :as contract-utils]))
+
+(deftest profile-set-bio-contract
+  (h/log-headline :contract/wakuext_setBio)
+  (rf-test/run-test-async
+   (h/with-app-initialized
+    (h/with-account
+     (contract-utils/call-rpc-endpoint
+      {:rpc-endpoint "wakuext_setBio"
+       :params       ["new bio"]
+       :on-error     #(is (nil? %))})
```

Comment:
just curious, if you change this to 
`:on-error #(is (not (nil? %))`
does the test fail?

Reply:
it will pas but it api throws any error then RPC will call on-error and test will fail